src/linear_algebra/multilinear_tensor_product.lean

```diff
@@ -0,0 +1,73 @@
+/-
```

Comment:
Can you make this `multilinear/tensor_product` and rename `multilinear` to `multilinear/basic`?